monai/utils/module.py

```diff
@@ -564,11 +566,11 @@ def version_leq(lhs: str, rhs: str) -> bool:
     """
 
     lhs, rhs = str(lhs), str(rhs)
-    pkging, has_ver = optional_import("pkg_resources", name="packaging")
+    has_ver = optional_import("packaging")
```

Comment:
Please change the code like this, and remove the import above and try to use `pkging.version` below.
```suggestion
    pkging, has_ver = optional_import("pkg_resources")
```